tests/integration/test_help.sh

```diff
@@ -16,27 +18,46 @@ IFS=$'\n\t'
 
 OUTPUT=$(/bin/true)
 
+# let's clean the environment first
+${ATOMIC} uninstall -f --storage=docker centos || true
+${ATOMIC} uninstall -f --storage=ostree centos || true
```

Comment:
Are you trying to delete images? Uninstall only handles "uninstall" labels for docker containers and removes the container for system containers, which shouldn't affect this test at all. I think what you want is "atomic images delete"?